static/maintenance/maintenance.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="theme-color" content="#ffffff">
+
+  <!-- Facebook Meta Tags -->
+  <meta property="og:url" content="https://robotornot.mainnet-beta.rarimo.com/">
+  <meta property="og:type" content="website">
+  <meta property="og:title" content="Prove Your Humanity | Rarimo">
+  <meta property="og:description" content="Prove your humanity using your identity provider of choice and get access to on-chain rewards with Rarimo.">
+  <meta property="og:image" content="../branding/og-image.png">
+  <meta property="og:locale" content="en_US">
+
+  <!-- Twitter Meta Tags -->
+  <meta name="twitter:card" content="summary_large_image">
+  <meta property="twitter:domain" content="robotornot.mainnet-beta.rarimo.com">
+  <meta property="twitter:url" content="https://robotornot.mainnet-beta.rarimo.com/">
+  <meta name="twitter:title" content="Prove Your Humanity | Rarimo'">
+  <meta name="twitter:description" content="Prove your humanity using your identity provider of choice and get access to on-chain rewards with Rarimo.">
+  <meta name="twitter:image" content="../branding/og-image.png">
+
+  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">  <link as="style" rel="stylesheet preload" type="text/css" href="/branding/fonts/index.css" crossorigin="anonymous">
+
+  <title>Prove Your Humanity | Rarimo</title>
+  <meta name="description" content="Prove your humanity using your identity provider of choice and get access to on-chain rewards with Rarimo.">
+
+  <link as="style" rel="stylesheet preload" type="text/css" href="../branding/fonts/index.css">
+  <link rel="icon" href="../branding/favicon.ico">
+  <link rel="icon" type="image/png" href="../branding/favicon.png">
+  <link rel="apple-touch-icon" href="../branding/apple-touch-icon.png">
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+      background-color: #F5F6FF;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      height: 100vh;
+      font-family: 'euclidCircularB';
```

Comment:
add fallback font `sans-serif`
```suggestion
      font-family: 'euclidCircularB', sans-serif;
```